packages/react-components/src/localization/locales/ar-SA/strings.json

```diff
@@ -343,4 +343,4 @@
     "rightNavButtonAriaLabel": "الصفحة التالية"
   },
   "AttendeeRole": "الحضور"
-}
\ No newline at end of file
+}

```

Comment:
Can we get the locale strings out of this PR?

Reply:
updated